plot/plot_figure.py

```diff
@@ -131,13 +133,29 @@ def plot_mosaic(config, mask, slice_dim=2, colormap=None, vspace=2, hspace=2, co
     """
     # Define variable h(height), w(width), and slices
     mask_shape = mask.shape[:3]
+
+    mask = mask[:, :, :, 1]
```

Comment:
why indexing here?

Reply:
I used it for plotting just the CT channel, but there is no problem if the line is deleted.